doc/Caliptra.md

```diff
@@ -181,58 +181,58 @@ Explicitly out of scope is how silicon integration into backend work is performe
 * Post manufacture test and initialization (OSAT)
 * Certification
 
-## Use Cases
+## Use cases
 
-The Silicon RoT use cases can be supported through the adoption of specific industry standards and association/consortium specifications. Refer to specific documents in [References](#References).
+The Silicon RoT use cases can be supported through the adoption of specific industry standards, and association and consortium specifications. For more information, see specific documents in [References](#References).
 
 In this version, Caliptra Silicon RoT desired capabilities address the basics of supply chain security use cases.
 
-### Supply Chain Security
+### Supply chain security
 
-* **Mutable Code Integrity:** The objective here is to prove the device is running genuine firmware that the device manufacturer can vouch for its authenticity and integrity, and the device owner can ensure only authorized updates are applied to the device. This flow is aligned with [Reference 10](#ref-10) and can be achieved with dual signature verification of equal imposition.
-* **Configuration and Lifecycle Management**: allow the platform owner to securely configure the RoT capabilities, and enable/authorize lifecycle state transitions of the SoC.
+* **Mutable code integrity:** The objective is to prove the device is running genuine firmware such that the device manufacturer can vouch for its authenticity and integrity, and the device owner can ensure only authorized updates are applied to the device. This flow is aligned with [Reference 10](#ref-10) and can be achieved with dual signature verification of equal imposition.
+* **Configuration and lifecycle management**: The objective is to allow the platform owner to securely configure the RoT capabilities, and to enable and authorize lifecycle state transitions of the SoC.
 
-### DICE Protection Environment
+### DICE protection environment
```

Comment:
Like "Security Protocol and Data Model", this should be capitalized - "DICE Protection Environment"

Reply:
fixed